main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -2350,7 +2351,7 @@ object Resolver {
       // Case 1: Exactly one match. Success.
       case caze :: _ => Result.Ok(caze)
       // Case 2: Multiple matches. Error
-      case cazes => throw InternalCompilerException(s"unexpected duplicate tag: '$qname'.", qname.loc)
+      case null => throw InternalCompilerException(s"unexpected duplicate tag: '$qname'.", qname.loc)
```

Comment:
Something is wrong here!

Reply:
The original case should have been: caze :: Nil not caze :: _

Reply:
(Sidenote: We never use null. )

Reply:
Hmm... true, but the Scala 3 compiler said that `cazes` was unreachable unless the match exp were `null`

Reply:
Here the original code is broken and should be fixed, i.e. the match before the wildcard should be caze :: Nil